website/docs/components/copy/snippet/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Copy Snippet
+description: A button that copies the text content of the button itself.
```

Comment:
ok, will work on making it more clear; also open to suggestions.

Reply:
Ok we talked about this and...that's what it does, though. Not sure what wording would would make this more clear, since it's explicitly stating what it does. Ideas?